lametro/models.py

```diff
@@ -911,7 +911,8 @@ class LAMetroSubject(models.Model):
         ('significant_date_exact', 'Significant Date'),
         ('motion_by_exact', 'Motion By'),
         ('topics_exact', 'Subject'),
-        ('plan_program_policy_exact', 'Plan, Program, or Policy')
+        ('plan_program_policy_exact', 'Plan, Program, or Policy'),
+        ('legislative_session', 'Legislative Session')
```

Comment:
Ditto here – legislative session is not a classification we pull from SES, so we don't need this.

Reply:
You might be able to remove the new migration when you remove this choice.